package.json

```diff
@@ -84,7 +84,7 @@
   "author": "Juan Cruz Viotti <juan@balena.io>",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.20.0 <13.0.0",
+    "node": ">=10.20.0 <15.0.0",
```

Comment:
I understand that, after this PR is merged, we still don't support Node.js v14 generally (e.g. `balena push` with devenv / BoB works with Node.js v10 and v12, but not v14). Therefore, this PR should not be amending this line. No?

Reply:
Basically let change that and see.

Reply:
If the Node.js v14 checks fail because of the warnings printed in connection with this `engines.node` restriction, we could either ignore it, or maybe hardcode exceptions for example here:
https://github.com/balena-io/balena-cli/blob/v12.42.0/tests/helpers.ts#L46-L52